database/migration/src/main/resources/changesets/enlarge_job_store_ids.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog objectQuotingStrategy="QUOTE_ALL_OBJECTS"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <!-- Drop the foreign key constraint from JOB_STORE_SIMPLETON_ENTRY to JOB_STORE_ENTRY to allow for the latter's PK
+         to be widened. -->
+    <changeSet author="mcovarr" id="drop_job_store_simpleton_entry_fk" dbms="mysql,hsqldb,postgresql,mariadb">
+        <dropForeignKeyConstraint
+                baseTableName="JOB_STORE_SIMPLETON_ENTRY"
+                constraintName="FK_JOB_STORE_SIMPLETON_ENTRY_JOB_STORE_ENTRY_ID"
+        />
+    </changeSet>
+
+    <!-- BEGIN JOB_STORE_ENTRY PK widening -->
+    <changeSet author="mcovarr" id="enlarge_job_store_entry_id" dbms="mysql,hsqldb,mariadb">
```

Comment:
Looks like for Posgresql there're 2 changesets: one for modifying table column type, and another one for altering the autoincrementing sequence, whereas for other DBs there's a single `addAUtoIncrement` changeset. Does this single changeset do all the work for those databases (updating a column and sequence at once)?

Reply:
That's right, the other databases can do the equivalent with a single changeset, as I previously learned [the hard way](https://github.com/broadinstitute/cromwell/blob/develop/database/migration/src/main/resources/changesets/enlarge_call_caching_hash_entry_id.xml#L6).